intro_to_r/intro_to_r.Rmd

```diff
@@ -239,9 +251,21 @@ of operations.  Here, we want to divide the number of boys by the total number
 of newborns, so we have to use parentheses.  Without them, R will first do the 
 division, then the addition, giving you something that is not a proportion.
 
+We can also pipe data into plots, as in
+```{r piped-plot}
+arbuthnot %>%
```

Comment:
Yes, I think the `.` (whatever we call it) should be left for much later. I had just seen @rudeboybert's comment about them and saw an opportunity to stick one in. Apologies for the ugly code :p